be/CMakeLists.txt

```diff
@@ -541,6 +544,7 @@ set(DORIS_DEPENDENCIES
     zstd
     arrow
     parquet
+    orc
```

Comment:
You'd better add this in next patch. If you do it this PR, this would make others fail, because they don't have ORC library.